### ISSUE TITLE
Installing neovim-git gets stuck at "Generating helptags ..."

### ISSUE THREAD

**Issue by akuntsch** [rmarquis/pacaur#325]
Hey, when trying to install neovim-git with pacaur the installation process always gets stuck at "Generating helptags ..." with CPU at 25% (which is one core running at 100% and it's cycling through the cores every few seconds).

The installation works fine when downloading the PKGBUILD manually and using makepkg.

It also seems to work fine with packer as AUR helper (see https://aur.archlinux.org/packages/neovim-git/ comments section).

Might be related to https://github.com/neovim/neovim/issues/2364.

**Comment by rmarquis** [comment]
Looks like a similar issue to #197.

**Comment by akuntsch** [comment]
Would the terminal output help you? It's pretty long though and I doubt it's helpful since it's just the compilation output which stops at "Generating helptags".

**Comment by rmarquis** [comment]
No, I can reproduce. This is the very same bug as #197, namely the piped 'yes' command conflicts with the check() routine. I am unsure how to fix this issue properly at this time. 

Cheap and very dirty workaround: add the following to your pacaur config file:
```
makeopts+=("--nocheck")
```

**Comment by akuntsch** [comment]
Doesn't work for me either.

**Comment by rmarquis** [comment]
Confirmed, the yes piping produces the helptag hanging.

However, even when running ```makepk -sfi``` (which is what pacaur uses), the package fails to install:

```
:: Proceed with installation? [Y/n] ==> WARNING: Failed to install built package(s).
```

The PKGBUILD does look correct, but there's definitely something weird going on here. No way pacaur can install this package if ```-sfi``` fails. If you can reproduce too, you might want to report to the AUR package maintainer.

**Comment by akuntsch** [comment]
I get the same error with `makepkg -sfi` and `makepkg -si`, however if I use `makepkg -s` or `makepkg -sf` and install manually with `pacman -U` it installs just fine.

**Comment by rmarquis** [comment]
Yes, but there are a lot of reason pacaur uses ```-i``` instead of using ```pacman -U``` to install created packages. If it's not supported by makepkg -i, it's not supported by pacaur. This should be fixed in the PKGBUILD (if faulty), otherwise fixed upstream. Nothing I can do here.

The hanging issue is however real.

**Comment by fwalch** [comment]
Hi, I'm the PKGBUILD maintainer for `neovim-git`. `makepkg -i` fails and the `check()` function doesn't make a difference, so I think it's an upstream issue..

Any idea what the underlying problem might be? I mean, why the installation fails? Is an extra <kbd>Return</kbd> pushed to the terminal, which then aborts the `Proceed with installation? [Y/n]` question?

**Comment by fwalch** [comment]
Looks like the problem was introduced upstream with merge of neovim/neovim#2025. I'm going to report upstream when I know the exact commit and how to reproduce it on a non-Arch system.

**Comment by rmarquis** [comment]
Thanks. I'll try to find a way to fix the pacaur specific issue, but I'm clueless about my available options - any idea welcome!

**Comment by tarruda** [comment]
Does removing the installation directory fixes the problem? See this for more info: https://github.com/neovim/neovim/issues/2364#issuecomment-91845325

**Comment by rmarquis** [comment]
@tarruda : doesn't seem so. Clean install with "makepkg -sfi", same issue.

**Comment by fwalch** [comment]
`makepkg -sfi` works now (neovim/neovim#2598). With `pacaur`, the installation still gets stuck at "Generating helptags", though.

**Comment by fwalch** [comment]
This should be fixed now (neovim/neovim#2801).

**Comment by rmarquis** [comment]
Thanks for the follow up!
